common/Helpers/RuntimeHelper.cs

```diff
@@ -2,22 +2,24 @@
 // Licensed under the MIT license.
 
 using System.Runtime.InteropServices;
-using System.Text;
 
 namespace DevHome.Common.Helpers;
 
-public class RuntimeHelper
+public static class RuntimeHelper
 {
-    [DllImport("kernel32.dll", CharSet = CharSet.Unicode, SetLastError = true)]
-    private static extern int GetCurrentPackageFullName(ref int packageFullNameLength, StringBuilder? packageFullName);
-
     public static bool IsMSIX
     {
         get
         {
-            var length = 0;
+            const int APPMODEL_ERROR_NO_PACKAGE = 15700;
+
+            uint length = 0;
+
+            return GetCurrentPackageFullName(ref length, null) != APPMODEL_ERROR_NO_PACKAGE;

```

Comment:
Can we add this method to NativeMethods.txt to benefit from the one generated by CsWin32?
```cs
PInvoke.GetCurrentPackageFullName(ref length, null) != WIN32_ERROR.APPMODEL_ERROR_NO_PACKAGE;
```